glustercli/cmd/volume-create.go

```diff
@@ -110,3 +128,19 @@ func volumeCreateCmdRun(cmd *cobra.Command, args []string) {
 	fmt.Printf("%s Volume created successfully\n", vol.Name)
 	fmt.Println("Volume ID: ", vol.ID)
 }
+
+func addThinArbiter(req *api.VolCreateReq, thinArbiter string) error {
+
+	s := strings.Split(thinArbiter, ":")
+	if len(s) != 2 && len(s) != 3 {
+		return fmt.Errorf("Thin arbiter brick must be of the form <host>:<brick> or <host>:<brick>:<port>")
+	}
+
+	// TODO: Handle this in a generic way, just like other volume set
+	// options that we're gonna allow later.
+	req.Options = map[string]string{
+		"afr.thin-arbiter": thinArbiter,
```

Comment:
Use "replicate.thin-arbiter" so that Volgen automatically adds to Volfile

Reply:
Looks like these options may become issue with afr prefix. None of the options added with afr prefix will appear in generated volfile under `type cluster/replicate`

Reply:
Okay. "Replicate" will work as the afr xlator is loaded under both (afr and replicate) names: https://github.com/gluster/glusterd2/blob/master/glusterd2/xlator/load.go#L169

Reply:
Not related to this patch, but other replicate options set accepts both `afr.*` and `replicate.*`, any suggestion to normalize these during Volume set?

Reply:
We can take a call on that later may be. `replicate` sounds more understandable though than `afr`. The same applies to `dht` vs `distribute`.